Testing/functional/tests/cypress/db/seed.sql

```diff
@@ -780,6 +780,27 @@ VALUES (
 	'System.DateTime, System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e', 
 	'"2021-01-20T00:00:00"'
 );
+INSERT INTO gateway."ResourceDelegate"(
```

Comment:
You need to update the integration/e2e/dashboard.cy.js test on lines 13 and 28.

Reply:
Thanks for this reminder!

![image](https://user-images.githubusercontent.com/19548348/228432403-c02f2630-7b40-4bcc-b357-e69e0580135c.png)